source.yml

```diff
@@ -91,6 +91,8 @@ version_control:
 overrides:
     - ^(orogen|typelib|rtt|utilrb|utilmm|rtt_typelib|tools/metaruby)$:
       branch: $ROCK_BRANCH
+    - rtt:
```

Comment:
Could you add a comment that this is temporary fix with link to the PR / reason why this is needed.
Otherwise approved from my side.

Reply:
Very good point ... Done.